site/docs/tools/spark_auth_config.md

```diff
@@ -0,0 +1,55 @@
+# Authentication in Spark
+
+When Nessie is used in Spark-based environments (either with [Iceberg](./iceberg/index.md) 
+or [Delta Lake](./deltalake/index.md)) the Nessie authentication settings are configured via Spark session properties.
+
+=== "Java"
+    ``` java
+    // local spark instance, assuming NONE authentication
+    conf.set("spark.sql.catalog.nessie", "org.apache.iceberg.spark.SparkCatalog")
+        .set("spark.sql.catalog.nessie.authentication.type", "NONE")
+        .set(...);
+    spark = SparkSession.builder()
+                        .master("local[2]")
+                        .config(conf)
+                        .getOrCreate();
+    ```
+=== "Python"
+    ``` python
+    # local spark instance, assuming NONE authentication
+    spark = SparkSession.builder \
+            .config("spark.sql.catalog.nessie", "org.apache.iceberg.spark.SparkCatalog") \
+            .config("spark.sql.catalog.nessie.authentication.type", "NONE") \
+            .config(...) 
+            .getOrCreate()
+    ```
+
+The `spark.sql.catalog.nessie` prefix identifies properties for the Nessie catalog. The `nessie` part is just
+the name of the catalog in this case (not to be confused with the Nessie project name).
+
+# Authentication Settings
+
+The value of the `authentication.type` property (with the appropriate prefix) can be one of the following:
+
+* NONE
+* BEARER
+* AWS
```

Comment:
Wonder whether we should mention `BASIC` here as well - but I'm not sure. Would that be useful for testing/evaluation purposes?

Reply:
Users will not be able to use `BASIC` unless they run a local Nessie Server build in `test` or `dev` mode. I do not think it is very helpful for users to have `BASIC` documented. If they want to have a realistic feel about how auth works in Nessie, they will have to use `BEARER` anyway because that is what we have in `prod`. As for playing with commits and CLI `NONE` is probably sufficient.